### DISCUSSION TITLE
Unusual Script Compilation Error

### DISCUSSION THREAD

**Discussion by pattdecc** [Aelto/W3_RandomEncounters_Tweaks#117] [Q&A] [answered]
Hey Aeltoth, anyway to solve this error? 
Error [modrandomencountersreworked]local\rer_scripts.min.ws(717): Global function 'reruninstall' is already defined. 

Pretty sure this one can't be simply solved by Script Merger. FYI, just updated to the 2.10.1 version. Thanks in advance!

**Comment by Aelto** [comment, marked as answer]
You should remove the uninstall script i shared on nexusmods, the command is now bundled in RER itself to make the uninstall process easier.

**Reply by pattdecc** [reply]
Hey Aeltoth, thanks for the fast reply! I tried deleting your modUninstallrer and the script compiled! However, the game turned out to be really lagging along with a constant ticking sound (like metal clashing) unless I disabled RER. Any advice to fix this? I really love this mod and it’d be a shame having to play without it, thanks!

**Reply by Aelto** [reply]
Yes, it is an issue that appears when updating from v2.8 to a newer version. Use the console command `rercompletecontract` and everything should be fine!

**Reply by pattdecc** [reply]
Okay. Just wanting to say that this mod is the main reason of my last 400 hrs playtime after campaign! Your work here really kept the W3 community alive and thriving. Thank you!

**Reply by Aelto** [reply]
Thank you, i really appreciate it! I'm happy to hear you've been using my mod for so long, and enjoyed it too. 400 hours, that's probably more than I've played it myself, incredible!
